msa/black-box-tests/src/test/resources/docker-compose.hybrid-test-extras.yml

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright © 2016-2023 The Thingsboard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+version: '2.2'
```

Comment:
we use `version: '3.0`

Reply:
Today I see the '2.2' in the docker submodule. Where can I find v '3'?

Reply:
You may find it here: https://github.com/thingsboard/thingsboard/blob/develop/3.5/docker/docker-compose.yml#L18

Reply:
We **DO NOT** use submodules for CE. To update submodules for PE: ```git submodule update –remote```

Reply:
Please fix the version, otherwise LGTM